build.psm1

```diff
@@ -696,19 +690,12 @@ function New-PSOptions {
     }
 
     if (-not $Runtime) {
-        $RuntimeFromDotNet = dotnet --info | ForEach-Object {
+        $Runtime = dotnet --info | ForEach-Object {
             if ($_ -match "RID") {
                 $_ -split "\s+" | Select-Object -Last 1
```

Comment:
FYI, Linux will require two packages because of packaging format differences (RPM vs DEB).